frontend/packages/dev-console/src/components/__tests__/ProjectsExistWrapper.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import ProjectsExistWrapper from '../ProjectsExistWrapper';
+
+const projects: any = {
```

Comment:
Can you create an interface for this object instead of using any?

Reply:
This should be the project prop using React.ComponentProps of the ProjectsExistsWrapper